authority-portal-frontend/src/app/common/components/form-elements/application-roles-tooltip/application-roles-tooltip.component.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2024 sovity GmbH
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Contributors:
+ *      sovity GmbH - initial implementation
+ */
+import {Component} from '@angular/core';
+
+@Component({
+  selector: 'app-application-roles-tooltip',
+  templateUrl: './application-roles-tooltip.component.html',
+})
+export class ApplicationRolesTooltipComponent {
+  // may need refinements
+  applicationRolesTooltip = `
+AUTHORITY USER
```

Comment:
Can we make the font bold and write it correctly? If not, then this is fine.
(Same for all other roles, don't want to repeat myself here)

Reply:
unfortunately no, only for the whole text